### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by vladchuk** [dart4e/dart4e#26] [General]
Hey, I just wanted to express my appreciation for dart4e and say thank for making it. It's nice to see that support and interest for Eclipse still exists. Once upon a time, there was a Dart plugin, Dartboard, which, unfortunately, was abandoned years ago.

Keep up the good work!

**Comment by sebthom** [comment]
Thanks a lot for your feedback! It's good to know that there are people still finding Eclipse plugins useful :-)
